libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Selectors/ConditionalSelector.cs

```diff
@@ -32,8 +32,8 @@ public class ConditionalSelector : ITriggerSelector
         [JsonProperty("condition")]
         public string Condition
         {
-            get { return condition?.ToString(); }
-            set { this.condition = (value != null) ? new ExpressionEngine().Parse(value) : null; }
+            get => condition?.ToString(); 
+            set => this.condition = (value != null) ? Parser.Parse(value) : null;
```

Comment:
>Parser [](start = 54, length = 6)

did something change?

Reply:
Using the parser...

---
In reply to: [360176260](https://github.com/microsoft/botbuilder-dotnet/pull/2936#discussion_r360176260) [](ancestors = 360176260)